pages/bookmarks.jsx

```diff
@@ -49,10 +49,10 @@ export default React.createClass({
     return(<p>Hi {user.username}! <button onClick={this.handleLogOutBtnClick} className="btn btn-link inline-link">Log out</button>.</p>);
   },
   getFailurePrompt() {
-    return (<p>Only Mozilla staff can login now as we test this new platform. Meanwhile, your favorited projects can still be remembered on this device.</p>);
+    return (<p>Something went wrong! Please refresh and try again.</p>);
```

Comment:
@Pomax since now all people with a Google account can sign in, will user who **does not** have a Google account ever be redirected back [to the original_url that we passed as the sign in url query](https://github.com/mozilla/network-pulse-api#get-loginoriginal_urlurl)? I'm assuming he/she will be "stuck" on the Google Sign In page after clicking the "sign in" link on Pulse. If that's the case we might not need this `getFailurePrompt()` and other relevant code anymore?